static/js/util/rest_test.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+import { assert } from "chai"
+
+import { anyProcessing, allLoaded, anyError, anyErrorExcept404 } from "./rest"
+
+describe("rest utils", () => {
+  describe("anyProcessing", () => {
+    it("should return false if nothing has processing true", () => {
```

Comment:
Can you add a test where `processing: false` and another object has `processing:true`?

Reply:
sure